### PR TITLE
fix installation of vsc-base 2.8.3 by switching to PythonBundle

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
installing this easyconfig fails because mock is automatically downloaded during the installation (a nasty side effect of the vsc-install-powered installation procedure used by vsc-base...)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2301.phanpy.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/ae74b6bf29b18ed84eb6587f13b0c1ea for a full test report.

**Comment by verdurin** [comment]
@boegel could you address the test complaints please?

File: easybuild/easyconfigs/v/vsc-base/vsc-base-2.8.3.eb [hunk 1]
```diff
@@ -9,17 +9,32 @@ description = """Basic Python libraries used by UGent's HPC group"""
...
-source_urls = [PYPI_SOURCE]
-sources = [SOURCE_TAR_GZ]
-checksums = ['9e102ca9d94ab97c2b974c63708dab7ea4dbaa3144787f06455e16445b92f204']
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
 
 # install as zipped egg to make sure that tools that use this as a dependency are responsive
 install_target = 'easy_install'
 zipped_egg = True
 
 download_dep_fail = True
```
Comment (branfosj) [inline easybuild/easyconfigs/v/vsc-base/vsc-base-2.8.3.eb:19]:
```suggestion
```

Should not be used in `PythonBundle`

File: easybuild/easyconfigs/v/vsc-base/vsc-base-2.8.3.eb [hunk 2]
```diff
@@ -9,17 +9,30 @@ description = """Basic Python libraries used by UGent's HPC group"""
...
 
 # install as zipped egg to make sure that tools that use this as a dependency are responsive
 install_target = 'easy_install'
 zipped_egg = True
 
-download_dep_fail = True
-
-options = {'modulename': 'vsc.utils'}
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
```
Comment (branfosj) [inline easybuild/easyconfigs/v/vsc-base/vsc-base-2.8.3.eb:19]:
```suggestion
```

Default not needed anymore

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0103u11a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A100-PCIE-40GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/549e4b9fb0f666836426e29604eabdae for a full test report.

**Comment by boegel** [comment]
not planning to pursue this, so closing...

**Pull request closed without merging**